test/screenshot/screenshot.js

```diff
@@ -1,60 +1,220 @@
+import {Readable} from 'stream';
+import {createHash} from 'crypto';
+import {readFile, writeFile} from 'fs';
+import {promisify} from 'util';
 import puppeteer from 'puppeteer';
 import compareImages from 'resemblejs/compareImages';
-import {promisify} from 'util';
-import {readFile, writeFile} from 'fs';
 import {assert} from 'chai';
+import Storage from '@google-cloud/storage';
 
 import comparisonOptions from './screenshot-comparison-options';
 
 const readFilePromise = promisify(readFile);
 const writeFilePromise = promisify(writeFile);
 
+const SERVICE_ACCOUNT_KEY = process.env.SERVICE_ACCOUNT_KEY;
```

Comment:
I would suggest prefixing all of our env vars with `MDC_`. This makes it easier to tell which part of the stack is using the var, and makes it easier to search/replace.

Also, we might want to rename this var to indicate that it's used by GCP.

E.g., `MDC_GCP_SERVICE_ACCOUNT_KEY` or `MDC_GCLOUD_SERVICE_ACCOUNT_KEY`.